package.json

```diff
@@ -1,19 +1,31 @@
 {
   "name": "adblock-rs",
-  "version": "0.2.9",
+  "version": "0.2.10",
   "description": "Very fast, Rust-based, native implementation of ad-blocker engine for Node",
   "main": "lib/index.js",
   "author": "Andrius Aucinas <aaucinas@brave.com>",
   "license": "MPL-2.0",
-  "homepage": "https://github.com/brave/adblock-rust#readme",
+  "homepage": "https://github.com/Lee182/adblock-rust#readme",
   "bugs": {
-    "url": "https://github.com/brave/adblock-rust/issues"
+    "url": "https://github.com/Lee182/adblock-rust/issues"
   },
-  "repository": "https://github.com/brave/adblock-rust.git",
+  "repository": "https://github.com/Lee182/adblock-rust.git",
```

Comment:
Could you remove the changes that aren't relevant to the issue? Otherwise I won't be able to accept this.